internal/venice-test-common/src/main/java/com/linkedin/venice/testng/VeniceSuiteListener.java

```diff
@@ -1,23 +0,0 @@
-package com.linkedin.venice.testng;
```

Comment:
`VeniceSuiteListener` looks to be used in `build.gradle` files.

Reply:
good catch! i will restore it then